operations/commit_queue.go

```diff
@@ -448,7 +448,12 @@ func backport() cli.Command {
 				return errors.Wrap(err, "uploading backport patch")
 			}
 
-			if err = patchParams.displayPatch(ac, backportPatch, uiV2, false); err != nil {
+			o := outputPatchParams{
```

Comment:
nit: can you call this params instead? `o` isn't a typical variable name (trying to figure out why I think this -- typically variables should be descriptive, so "params" is closer, or if it's going to be abbreviated  down to one letter, it should be on the noun, not the descriptive word so "p" is closer than "o")

Reply:
My only concern was that in this particular file we call the 'patchParams' as 'params'- But the usage is pretty small so I actually think it won't be confusing- i'll switch it

Reply:
in that case we could go so far as to say outputParams for the variable -- better to have clarity than keep it short